.github/workflows/ci-build-and-test.yml

```
@@ -30,6 +30,9 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
+      - name: Run commitlint
+        run: npx commitlint --from=origin/main --to=HEAD
```

I don't think you need the `=`
```suggestion
        run: npx commitlint --from origin/main --to HEAD
```
https://commitlint.js.org/guides/ci-setup.html#github-actions